### ISSUE TITLE
Company logo in menu

### ISSUE THREAD

**Issue by dechkovanen** [macadmins/SupportCompanion#63]
If company logo is set, it could be shown instead of "Support Companion" title in the menu, or alternatively in lower left corner of the menu with Quit button in lower right corner.

**Comment by almenscorner** [comment]
How it works at the moment is that if you set `BrandName` to an empty string, i.e., `<string/>`, it will not show anything but the logo

**Comment by dechkovanen** [comment]
Yes, it does - in the app. In the new popdown menu it does not show the company logo. The menu could reflect the settings in the app and show the logo in the popdown menu as well.

**Comment by almenscorner** [comment]
Question is, should the logo show in the menu or should it be kept as lightweight as possible?

**Comment by dechkovanen** [comment]
Also leaving the `BrandName` empty leaves the extra whitespace in the top of the menu. My suggestion is remove whitespace if there's no BrandName set and display the logo at bottom left corner and resized to appropriate maximum size (in case the logo would be huge otherwise). I don't see small('ish) logo as unnecessary clutter. And you can always include a `HideLogoInMenu` or such boolean in config for those who don't want it there.

**Comment by almenscorner** [comment]
Okay, I will add in the next release so that the logo is shown with a max width of 150 and scales to fit. The logo will be displayed where `BrandName` is and if you do not want the logo in the tray menu it can be hidden and only keep `BrandName` if need be

**Comment by almenscorner** [comment]
example:
<img width="525" alt="image" src="https://github.com/user-attachments/assets/333d079f-2aee-48bf-8d1e-a91b6a81348c">
